private/bufpkg/bufcheck/buflint/buflint_test.go

```diff
@@ -1109,30 +1168,40 @@ func testLintWithOptions(
 		storageos.ReadWriteBucketWithSymlinksIfSupported(),
 	)
 	require.NoError(t, err)
+	var options []bufworkspace.WorkspaceBucketOption
```

Comment:
This isnt used

Reply:
Ah yes, I ended up going with targeting the opaque IDs instead